resolver_limited_test.go

```diff
@@ -87,10 +87,22 @@ func TestLimitedResolver(t *testing.T) {
 		}
 	}
 	{
-		r := NewLimitedResolver(testResolver, 2, 2)
+		r := NewLimitedResolver(testResolver, 2, 2, 2)
 		b, _, err := r.MatchMX("mxmustfail.", newMatcher(net.ParseIP("10.0.0.10")))
 		if b || err != ErrDNSLimitExceeded {
 			t.Errorf("MatchMX got: %v, %v; want false, ErrDNSLimitExceeded", b, err)
 		}
 	}
+	{
+		dns.HandleFunc("void.test.", Zone(map[uint16][]string{}))
+		defer dns.HandleRemove("void.test.")
+
+		r := NewLimitedResolver(testResolver, 6, 6, 2)
+		_, _, _ = r.LookupTXTStrict("void.test.")
+		_, _, _ = r.LookupTXTStrict("void.test.")
+		_, _, err := r.LookupTXTStrict("void.test.")
+		if err != ErrDNSVoidLookupLimitExceeded {
+			t.Errorf("LookupTXTStrict got: %v; want ErrDNSVoidLookupLimitExceeded", err)
+		}
```

Comment:
It does not check if resolver allowed to do first 2 lookups

Reply:
done